arch/xtensa/include/xtensa_asm2_s.h

```diff
@@ -371,7 +371,7 @@ _xstack_call0_\@:
 	entry a1, 16
 	mov a1, a2
 	rsr.ZSR_EPS a2
-	wsr.PS a2
+	wsr.ps a2
```

Comment:
Why are "rsr" and "wsr" two different patches? :)

Reply:
That wound up getting handled separately. It was an artefact of my workflow--no real reason why could not have been squashed.